pkg/pusher/pusher.go

```diff
@@ -163,6 +167,13 @@ LOOP:
 						return
 					}
 				}
+
+				_, err = crypto.Recover(receipt.Signature, receipt.Address.Bytes())
+				if err != nil {
+					err = fmt.Errorf("pusher receipt recover: %w", err)
```

Comment:
The message should have a colon after the `pusher` - `"pusher: receipt recover: %w"`